app/src/main/res/layout/include_find_in_page.xml

```diff
@@ -0,0 +1,113 @@
+<?xml version="1.0" encoding="utf-8"?><!--
+  ~ Copyright (c) 2018 DuckDuckGo
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/findInPageContainer"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/brownishGray"
+    android:clickable="true"
+    tools:ignore="KeyboardInaccessibleWidget"
+    tools:layout_height="?attr/actionBarSize">
+
+    <EditText
+        android:id="@+id/findInPageInput"
```

Comment:
Can we make it more consistent with the omnibar? I think these options work:
`android:background="@color/colorPrimary"`
`android:fontFamily="sans-serif-medium"`
`android:textSize="13sp"`

Once we've released to beta let's ask design to play with it too to see if they have any feedback.